modules/exploits/windows/http/sepm_auth_bypass_rce.rb

```diff
@@ -0,0 +1,110 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Symantec Endpoint Protection Manager Auth Bypass and RCE",
+      'Description'    => %q{
+      This module exploits two separate vulnerabilities in Symantec Endpoint Protection Manager
+      in order to achieve a remote shell on the box.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'bperry', #metasploit module
+          'CodeWhiteSec' #discovery
```

Comment:
It’s not that clear from neither the blog post nor the tweet. But the Symantec advisory SYM05-007 and the CVEs credit me.